simulator/infra/types.h

```diff
@@ -47,9 +47,13 @@ using uint128 = unsigned __int128;
 
 static inline std::ostream& operator<<(std::ostream& out, uint128 value)
 {
-    if ((out.flags() & std::ios::hex) != 0)
```

Comment:
Wait a second — what happened with hexadecimal output?

Reply:
I've removed it by accident. So it should be also tested to prevent such accidents?

Reply:
Yes.